ansible/templates/minigraph_cpg.j2

```diff
@@ -112,16 +132,14 @@
           </BGPPeer>
 {% endif %}
 {% endfor %}
-{% if switch_type is defined and switch_type == 'voq' %}
-{% for all_idx in range(all_inbands|length) %}
-{% if voq_inband_ip != all_inbands[all_idx] %}
+{% if num_asics == 1 and switch_type is defined and switch_type == 'voq' %}
```

Comment:
why do we need num_asics == 1 condition as well here ?

Reply:
Because we are in BgpRouterDeclaration of the whole linecard (inventory_hostname). If I am single asic linecard, then I need to add all my voq peers to the other linecards. For multiasic, the voq peers go in the asisc BgpRouterDeclaration